File: codeup/100000576/D.py
#!/usr/bin/python3
import sys

while True:
    line = sys.stdin.readline()
    if not line:
        break
    N = int(line)
    num_list = [int(x) for x in input().split()]

    M = int(input())
    for num in input().split():
        find_flag = False
        for x in num_list:
            if x == int(num):
                print("YES")
                find_flag = True
                break
        if not find_flag:
            print("NO")

File: codeup/100000576/D.cpp
#include <iostream>
using namespace std;

int main() {
  int N, M, i, j, num, num_list[102];

  while (scanf("%d", &N) != EOF) {
    for (i = 0; i < N; i++) {
      scanf("%d", &num_list[i]);
    }
    scanf("%d", &M);
    for (i = 0; i < M; i++) {
      scanf("%d", &num);
      for (j = 0; j < N; j++) {
        if (num == num_list[j]) break;
      }
      if (j < N)
        printf("YES\n");
      else
        printf("NO\n");
    }
  }

  return 0;
}
